variables.env

```diff
@@ -0,0 +1,5 @@
+
+DIFFDOCK_CACHE=~/.cache/nim
+MOLMIM_CACHE=~/.cache/nim
+ALPHAFOLD2_CACHE=~/.cache/nim
+AI_WORKBENCH_FLAG=true
```

Comment:
Will this cause issues in non-workbench environments? It looks like it toggles the URL used for NIM paths in the ipython notebook - if this is set in a non-workbench environment, will it still work properly?

Reply:
that is a great question.

workbench specifically modifies and invokes this file to load environment variables.

the only way it would impact a non-workbench environment is if the user would go out of their way to use it to define those environment variables.

so i think the answer is "yes, a non-workbench env will work properly even if this file is there."

but we should test it.

Reply:
@edawson - my understanding is that you can pull the workbench repo (i.e. your's after the PR) and get it running manually with no issues, and where there's no invocation of anything workbench related.

@nv-edwli - can you verify this?

Reply:
![image](https://github.com/user-attachments/assets/ab43e64b-767f-43d1-83be-d4f336080948)
I have verified that the blueprint runs outside of AI Workbench using standard ``docker compose up`` + ``jupyter notebook``. 

This makes sense because AI Workbench specifically invokes the ``variables.env`` file to set environment variables for the project container. The flag in this file toggles between the two environments in the first notebook cell and enables the proper routing for AIWB users. 

If running outside of AI Workbench (just following the standard instructions in the README), the user is not invoking the ``variables.env`` file at any point and so the flag environment variable does not make it to the notebook environment. So it should still work as it did prior to adding AI Workbench support.